Index: binutils-mingw-w64/upstream/libiberty/fibheap.c
===================================================================
--- binutils-mingw-w64.orig/upstream/libiberty/fibheap.c
+++ binutils-mingw-w64/upstream/libiberty/fibheap.c
@@ -22,9 +22,6 @@ Boston, MA 02110-1301, USA.  */
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
-#ifdef HAVE_LIMITS_H
-#include <limits.h>
-#endif
 #ifdef HAVE_STDLIB_H
 #include <stdlib.h>
 #endif
@@ -33,6 +30,7 @@ Boston, MA 02110-1301, USA.  */
 #endif
 #include "libiberty.h"
 #include "fibheap.h"
+#include <limits.h>
 
 
 #define FIBHEAPKEY_MIN	LONG_MIN
Index: binutils-mingw-w64/upstream/libiberty/pex-unix.c
===================================================================
--- binutils-mingw-w64.orig/upstream/libiberty/pex-unix.c
+++ binutils-mingw-w64/upstream/libiberty/pex-unix.c
@@ -27,6 +27,7 @@ Boston, MA 02110-1301, USA.  */
 #include <stdio.h>
 #include <signal.h>
 #include <errno.h>
+#include <fcntl.h>
 #ifdef NEED_DECLARATION_ERRNO
 extern int errno;
 #endif
@@ -42,9 +43,6 @@ extern int errno;
 
 #include <sys/types.h>
 
-#ifdef HAVE_FCNTL_H
-#include <fcntl.h>
-#endif
 #ifdef HAVE_SYS_WAIT_H
 #include <sys/wait.h>
 #endif
